sources/communityLib.move

```diff
@@ -241,6 +241,26 @@ module peeranha::communityLib {
         event::emit(UnfreezeCommunityEvent {userId: userId, communityId: communityId});
     }
 
+    public entry fun initPermistions(   // make universal?
```

Comment:
do we really need this action?
Can we run all these from a script outside of contract?